packages/cyberstorm/src/components/TextInput.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { MouseEventHandler, PropsWithChildren, ReactNode } from "react";
+import styles from "./componentStyles/TextInput.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircleX } from "@fortawesome/pro-regular-svg-icons";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+
+export interface TextInputProps {
+  onChange?(event: React.ChangeEvent<HTMLInputElement>): void;
```

Comment:
Should `value` and `onChange` perhaps be required props? What use is a text input if you can't change the input?

Reply:
Yea they should, good point out 👍